src/app_service/service/chat/service.nim

```diff
@@ -433,6 +434,7 @@ QtObject:
         slot: "onAsyncSendImagesDone",
         chatId: chatId,
         imagePathsJson: imagePathsJson,
+        tempDir: TMPDIR,
```

Comment:
Not sure this is gonna work on mobile, just saying :)

Reply:
But it doesn't work at all atm, so it's something that will need to be tackled separately anyway I guess